packages/@react-aria/virtualizer/src/ScrollView.tsx

```diff
@@ -120,6 +122,7 @@ export function useScrollView(props: ScrollViewProps, ref: RefObject<HTMLElement
           setScrolling(false);
           state.scrollTimeout = null;
 
+          window.dispatchEvent(new Event('tk.connect-observer'));
```

Comment:
Need to clean this up in the return of an effect, otherwise we might end up permanently disconnected